radixdlt-core/radixdlt/src/main/java/org/radix/network/messaging/Message.java

```diff
@@ -96,7 +94,7 @@ protected Message() {
 		this(Time.currentTimestamp());

```

Comment:
Seems it will make this class untestable

Reply:
It was not testable before and, given that this is an abstract class, I see no problem with that.